S/SDPLR/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SDPLR"
+version = v"1.0.3"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://sburer.github.io/files/SDPLR-1.03-beta.zip", "f1f945734f72e008fd7be8544b27341b179292c3304226563d9c0f6cf503b2eb")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/SDPLR*
+make LAPACK_LIB=-lopenblas BLAS_LIB=
+cp sdplr $prefix/sdplr
```

Comment:
Please use verbose commands when copying stuff around, to keep record of what's happening (you don't know how many things have been copied incorrectly). `install` is useful for ensuring the mode of the installed files and that the destination directory is created automatically if needed
```suggestion
install -Dvm 755 "sdplr${exeext}" "${bindir}/sdplr${exeext}"
```

Reply:
Thanks! Will do next time